src/main/java/com/insightfullogic/honest_profiler/core/sources/VirtualMachine.java

```diff
@@ -63,6 +66,19 @@ public boolean isAgentLoaded()
         return agentLoaded;
     }
 
+    public LogSource getLogSourceFromVmArgs()
+    {
+        int agentPathStart = vmArgs.indexOf("-agentpath") + "-agentpath".length();
+        int agentPathEnd = vmArgs.indexOf(' ', agentPathStart);
+        String agentPath = vmArgs.substring(agentPathStart, agentPathEnd);
+        int logPathStart = agentPath.indexOf("logPath=") + "logPath=".length();
+        int commaPos = agentPath.indexOf(",", logPathStart);
+        int spacePos = agentPath.indexOf(' ', logPathStart);
+        int logPathEnd = commaPos > 0 ? commaPos : (spacePos > 0 ? spacePos : agentPath.length());
+
+        return new FileLogSource(new File(agentPath.substring(logPathStart, logPathEnd)));
+    }
```

Comment:
No,
previous version seems only used [getLogSource()](https://github.com/RichardWarburton/honest-profiler/blob/master/src/main/java/com/insightfullogic/honest_profiler/core/sources/VirtualMachine.java#L66-L82) to get latest log file at userDir
And current version used [getLogSourceFromVmArgs()](https://github.com/PhRX/honest-profiler-fx-ui/blob/master/src/main/java/com/insightfullogic/honest_profiler/core/sources/VirtualMachine.java#L69-L80) depends on vmArgs